articubench/util.py

```diff
@@ -795,3 +799,36 @@ def get_tube_info_stepwise(tube_length, tube_area, steps = [5, 8, 11, 13, 14, 15
                 raise Exception("calculate must be one of ['raw', 'mean', 'binary']")
         section_per_time += [section]
     return section_per_time
+
+
+def download_pretrained_weights(*, skip_if_exists=True, verbose=True):
+    package_path = DIR
+    model_weights_path = os.path.join(package_path, 'models')
+    if os.path.isdir(model_weights_path):
```

Comment:
always true, since we have model_weights_path in our current implementation as an empty directory. now if we also skip if exists, we won't download anything. Perhaps add something like:
not any(os.scandir(directory)) 
to check for empty directory